### ISSUE TITLE
current_user.is_authenticated() is not valid in Flask-Login==0.3.2

### ISSUE THREAD

**Issue by bilalel** [imwilsonxu/fbone#45]
Hello,

In Flask-Login==0.3.2, is_authenticated isn't a method anymore but a boolean.
Is it pythonic if we replace : 
```
if current_user.is_authenticated():
    pass
```
by :
```
if current_user.is_authenticated:
    pass
```
If you think it's a good workaround, I can send you a pull request.

Br,
Bilal

**Comment by jpeeler** [comment]
It happened a long time ago in 0.3.0 too. I'll submit a PR since maybe it'll save somebody else from having to do it.

**Comment by imwilsonxu** [comment]
Fix with 6ff899e7e8f8d5de7e55a0b8cb1ec749192798d7
